apps/blog/src/api/services/auth/Cookie.service.ts

```
@@ -0,0 +1,61 @@
+    });
+  }
+
+  public clear({ res }: { res: NextResponse }) {
+    res.cookies.delete(COOKIE_NAMES.CSRF_TOKEN);
+  }
+}
+
+export class CookieService {
+  public static session = new SessionCookieService();
+  public static csrf = new CSRFCookieService();
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)